core/pkg/registryadaptors/gcp/v1/Readme.md

```diff
@@ -0,0 +1,26 @@
+# GCP Adaptor
+
+### How we add gcp adaptor
+
+As there can be possiblities of use of multiple registries we check for each adaptor if we have required credentias. For every adaptor having credentials we append the adaptor to the adaptors slice.
+
+Particularly for gcp, we frstly bring the `gcpCloudAPI` from the connector. We still haven't created a proper function that initiats the gcpCloudAPI with projectId, credentialsPath, credentialsCheck fields. We check for `credentialsCheck` bool which is set true when we have credentials(to be set when initializing the gcpCloudAPI) 
```

Comment:
It still not clear to me how a user can make use of the GCP adaptor, is it something you're still working on ?

Reply:
if one has a image in the GCR registery then the adaptor will fetch data from the registery automatically 

More Info: 
We create an array of adaptors over here https://github.com/kubescape/kubescape/pull/825/files#diff-ae5ac0a40a815afd92d29c49fa4bc141c196fef53c4a9c9f9da927760f72661bR151 and so based on the image present respective adaptors come in action and fetch data